server/src/main/java/org/elasticsearch/cluster/metadata/DiffableStringMap.java

```diff
@@ -39,15 +39,18 @@
  */
 public class DiffableStringMap extends AbstractMap<String, String> implements Diffable<DiffableStringMap> {
 
+    public static final DiffableStringMap EMPTY = new DiffableStringMap(Collections.emptyMap());
```

Comment:
This is not part of linked patch, ES introduced it inhttps://github.com/elastic/elasticsearch/commit/5e17bc5dcc52b1c3dc92e51a6f154547aef3d24a#diff-75af5216d9a2e2449c577ad5358bb634f21eb43d325035543b099e03cc958d5dR42 (which I don't think we need to port, so took only this specific variable)